src/shared/Map/BiomassMap.jsx

```diff
@@ -31,6 +31,8 @@ const BiomassMapComp = ({ variant }) => {
   const [features, setFeatures] = useState(mapPolygon);
   const [drawEvent, setDrawEvent] = useState({});
 
+  const mapboxToken = import.meta.env.VITE_MAPBOX_TOKEN;
```

Comment:
can you make a keys.js file similar to the selector and then import the env files from there to be consisten with other tools?

Reply:
Yes, such changes have been made in the user history pr